Gemfile

```diff
@@ -1,6 +1,16 @@
-source "https://rubygems.org"
+source 'https://rubygems.org'
 
-git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
+git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
 # Specify your gem's dependencies in site_search.gemspec
 gemspec
+
+group :development, :test do
+  gem 'dotenv'
+  gem 'pry'
```

Comment:
I think this needs to be required somewhere for it to load. I had to require it in spec_helper for example